scripts/test_antlr_grammar.sh

```diff
@@ -116,7 +116,9 @@ done < <(
   grep -riL -E \
     "^\/\/ (Syntax|Type|Declaration)Error|^\/\/ ParserError (2837|3716|3997|5333|6275|6281|6933|7319)|^==== Source:" \
     "${ROOT_DIR}/test/libsolidity/syntaxTests" \
-    "${ROOT_DIR}/test/libsolidity/semanticTests" \
+    "${ROOT_DIR}/test/libsolidity/semanticTests" |
+  grep -v -E 'comments/.*_unicode.*.sol'
+  # Skipping the unicode tests as I couldn't adapt the lexical grammar to recursively counting RLO/LRO/PDF's.
```

Comment:
On a new line this looks like an independent command. Maybe it would be clearer indented?

Also, I'd keep the exclusions as narrow as possible.

```suggestion
    "${ROOT_DIR}/test/libsolidity/semanticTests" |
      grep -v -E 'comments/.*_unicode_direction_override.*.sol'
      # Skipping the unicode tests as I couldn't adapt the lexical grammar to recursively counting RLO/LRO/PDF's.
```